tests/unit/clustering/kmeans_unittest.cc

```diff
@@ -168,6 +168,35 @@ TEST(KMeans, minibatch_training_test)
 	}
 }
 
+TEST(KMeans, minibatch_fixed_centers)
+{
+
+	SGMatrix<float64_t> X{{2, 1}, {4, 1}, {0, 1}, {2, 4}, {0, 4}, {4, 4},
+	                      {5, 4}, {1, 0}, {2, 2}, {2, 3}, {5, 5}, {-1, 1}};
+
+	SGMatrix<float64_t> initial_centers(2, 2);
+	initial_centers(0, 0) = 0;
+	initial_centers(1, 0) = 0;
+
+	initial_centers(0, 1) = 1;
+	initial_centers(1, 1) = 1;
```

Comment:
you could write this a one liner `SGMatrix<float64_t> initial_centers{{0, 1}, {0, 1}}` no?

Reply:
done